Common/src/linear_algebra/CSysSolve_b.cpp

```diff
@@ -25,6 +25,7 @@
  * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "../../include/CConfig.hpp"
```

Comment:
Why did you add that include?

Reply:
I moved it from the hpp, maybe it is not needed at all, I'll check.

Reply:
Ah I didn't see that :) yep, then its definitely needed.

Reply:
Actually only needed the datatype include not the full config.